components/dashboards-web-component/src/utils/DashboardReportGenerator.js

```diff
@@ -281,7 +289,7 @@ export default class DashboardReportGenerator {
         const reportName = widgetName + '.pdf';
 
         DashboardAPI.getDashboardReportPdfConfigs().then((res) => {
-            if (res.data !== '') {
+            if (JSON.stringify(res.data) !== JSON.stringify({})) {
```

Comment:
Incorrect way of checking object emptiness. Lets use lodash `isEmpty` function.
https://lodash.com/docs#isEmpty